BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -102,6 +103,63 @@ public boolean onOptionsItemSelected(MenuItem item) {
     return super.onOptionsItemSelected(item);
   }
 
+  private void handleBackupLibrary() {
+    setStoragePermission();
```

Comment:
Why do you use a single function in that function? I think one function is sufficient for that.